roles/testconfig/tasks/main.yml

```diff
@@ -0,0 +1,143 @@
+---
+#- include_vars:
+#    file: images.yml
+#    name: images
+
+- name: source the overcloudrc file
+  shell: source /home/stack/overcloudrc
+
+- include: auth_old.yml
```

Comment:
This include will not work for you in case auth_old.yaml file still located at the root of the repo.

Reply:
It worked for me, where do you suggest to place it then?

Reply:
Reply in the comment below.